### ISSUE TITLE
Clock with the current weather icon overlay

### ISSUE THREAD

**Issue by vednolacni** [outadoc/immich-home-assistant#14]
Is it possible to make something like this?

![slika](https://github.com/user-attachments/assets/70852b5d-e997-40b6-82a6-89570db6e00a)

**Comment by sdholden28** [comment]
Yes, but that's not what this integration is for. You would use WallPanel or something similar for that. Immich would just provide the static background images. https://community.home-assistant.io/t/wallpanel-addon-wall-panel-mode-for-your-home-assistant-dashboards/449857
